src/ticks/disassembler_main.c

```diff
@@ -49,13 +49,13 @@ int main(int argc, char **argv)
     char  *program = argv[0];
     char  *filename;
     char  *endp;
-    uint16_t    org = 0;
-    int         start = -1;
-    uint16_t    end = 65535;
+    unsigned int    org = 0;
+    int    start = -1;
+    unsigned int    end = -1;   // Max value (all f's)
```

Comment:
This seems a bit fishy and playing around with limits and type conversion.

If it were INT_MAX then would we just disassemble the memory block 2^16 times?

Reply:
Probably UINT_MAX since *end* is unsigned?

Reply:
The function that does the disassembly takes a signed int I think?

Reply:
It does. I was following the original code which defined *end* as unsigned. I'll change it to signed and test it.

Reply:
Updated and tested.